artichoke-backend/Cargo.toml

```diff
@@ -52,3 +52,4 @@ artichoke-array = []
 artichoke-debug = ["backtrace"]
 artichoke-random = ["rand"]
 artichoke-system-environ = []
+artichoke-all-converter = []
```

Comment:
can you alphabetize this list?

Reply:
based on the flag used in the code, it looks like this should be

```toml
artichoke-all-converters = []
```

converters with the trailing `s`.